packages/countries/src/lazy-country-symbol/LazyCountrySymbol.tsx

```diff
@@ -0,0 +1,36 @@
+import { Suspense, useEffect, useState, useTransition } from "react";
```

Comment:
Just to add what we've discussed here:

We should be able to simplify this by moving the Suspense component to be on the user side.
We can't use setTransition but we should be able to just use Line 23 directly with the prop